python/cudf/cudf/core/column/datetime.py

```diff
@@ -525,7 +552,14 @@ def infer_format(element: str, **kwargs) -> str:
     fmt = _guess_datetime_format(element, **kwargs)
 
     if fmt is not None:
-        return fmt
+        if ".%f" in fmt:
+            pass
+            # PANDAS BUG: https://github.com/pandas-dev/pandas/issues/52418
```

Comment:
Given the response in https://github.com/pandas-dev/pandas/issues/52418#issuecomment-1497565169, how is this format later used?

Reply:
The underlying `time` library in c++ expects `.%3f, .%6f & .%9f.`

Reply:
Got it. Yeah I don't it will be likely for `get_datetime_format` to support the number of digits in the microseconds